war/.gitignore

```diff
@@ -1,2 +1,10 @@
-work
+work
```

Comment:
I thought that war module existed for building war. Shouldn't all UI trash go to separate ui maven module?